erpnext/stock/doctype/delivery_note/test_delivery_note.py

```diff
@@ -486,7 +486,7 @@ def test_dn_billing_status_case2(self):
 
 		self.assertEqual(dn2.get("items")[0].billed_amt, 400)
 		self.assertEqual(dn2.per_billed, 100)
-		self.assertEqual(dn2.status, "Completed")
+		self.assertEqual(dn2.status, "To Deliver")
```

Comment:
I'm not sure about this, why is a DN that was supposed to be completed now saying "To Deliver"?

Reply:
@neilLasrado yes, so initially Delivery Note Status was updated on the bases of Sales and the payment made against it. Now we have added "Delivered" check box in Delivery Note. This is where status is updated to "To Deliver", as "Delivered" check will always remain unchecked when the Note is submitted

Reply:
Have you added a test case to check if the DN gets marked as completed if the delivered checkbox is checked?